utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java

```diff
@@ -11,6 +11,7 @@
 import static utam.core.selenium.element.DriverAdapter.getNotFoundErr;
 import static utam.core.selenium.element.DriverAdapter.getSeleniumDriver;
 
+import java.time.Duration;
```

Comment:
redundant import

Reply:
This one is required; `clickAndHold` uses it to change the integer number of seconds into a `Duration` of seconds for how long to hold the click.